test/bats/alibabacloud.bats

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bats
```

Comment:
Checkout https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/main/test/bats/akeyless.bats which was added recently. I think we only need basic tests for validating the provider works the defined RPC interface and don't need extensive tests like rotation or sync as k8s secrets. The feature specific tests for the driver are already run with e2e-provider and doesn't need to be part of every provider.

Reply:
Have removed the extensive cases, and only basic test are retained, please review again.